src/components/molecules/TrashedList/index.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <div class="trashed-list">
+    <app-heading :level="1">{{ trashedTitle }}</app-heading>
+    <app-router-link
+      :to="path"
+      key-color
+      white
+      bg-lightgreen
+      small
+      round
+      hover-opacity
+      class="trashed-list__link"
+    >
+      すべての記事一覧へ戻る
+    </app-router-link>
+    <table class="trashed-list__table">
+      <thead class="trashed-list__table-head">
+        <tr>
+          <th v-for="(thead, index) in theads" :key="index">
+            <app-text tag="span" theme-color bold>
+              {{ thead }}
+            </app-text>
+          </th>
+        </tr>
+      </thead>
+      <transition-group
+        name="fade"
+        tag="tbody"
+        class="trashed-list__table-body"
+      >
+        <tr v-for="trash in targetArray" :key="trash.id">
```

Comment:
[imo]

`targetArray` という変数しか使えないといったような特別な理由がない限り、配列の変数名は「対象の配列がどんな配列か」がわかる英単語の複数形にした方が見通しが良いです。

```suggestion
        <tr v-for="trash in trashes" :key="trash.id">
```